src/glaciers/data/surfacevelocitydata_utils.jl

```diff
@@ -0,0 +1,105 @@
+export initialize_surfacevelocitydata
+export plot_timeseries, plot_count
+
+"""
+    initialize_surfacevelocitydata(file::String; interp=false)
+
+Initialize SurfaceVelocityData from Rabatel et. al (2023). 
+
+Arguments
+=================
+    - `file`: name of netcdf file with data
+    - `interp`: boolean variable indicating if we use the inporpolated data or not
+"""
+function initialize_surfacevelocitydata(file::String; interp=false)
+
+    # Date of first adquisition
+    date1 = ncread(file, "date1")
+    # Date of second adquisition
+    date2 = ncread(file, "date2")
+
+    # Middle date
+    if !interp 
+        date_mean = 0.5 .* date1 .+ 0.5 .* date2
+        date_mean_offset = date1_offset = date2_offset = 0
+    else 
+        date_mean = ncread(file, "mid_date")
+        # Extract string from metadata of when days start counting
+        # Default format of attribute "units" is "days since 2015-08-14 00:00:00"
+        date_mean_since = ncgetatt(file, "mid_date", "units")[12:21] # e.g., "2015-08-14"
+        date1_offset_since = ncgetatt(file, "date1", "units")[12:21] # e.g., "2015-07-30"
+        date2_offset_since = ncgetatt(file, "date2", "units")[12:21] # e.g., "2015-08-29"
+        # Convertion to Julia datetime
+        date_mean_offset = datetime2julian(DateTime(date_mean_since)) - 2400000.5
```

Comment:
Why do we still need to convert to Julian time? This was used before when we had to interact with Python dates, but this should no longer be the case, right?

Reply:
I am not sure what you are referring @JordiBolibar , but this transformation is required since that is the original format of the dataset. Days are count from customize starting dates, so this is necessary to set all in the same time reference. 

Anyways... this is just for the intepolated dataset, I would not pay much attention to this for now.